readme.md

```diff
@@ -1,10 +1,29 @@
 # ovos-PHAL-plugin - alsa volume control
 
-controls system volume with alsa
+## Install
+
+```sh
+python -m pip install git+https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/OpenVoiceOS/ovos-PHAL-plugin-alsa
```

Comment:
Ok, I will put the 3 examples, with explanation.
- with user token, because after a while github blocks you from downloading. if you want to upgrade all your skills you have to use a token
- simple pip install and from source the user needs to be aware that he will be installing different versions.

this is important especially now that you have enabled osm to install plugins too, they are installed in another location and you (at least me) don' t realize what is happening and you have 2 different version of the same skill/plugin in two different places

Reply:
@denics could you please resolve the review, so we can merge?

Reply:
ok, I'll come back to this in about one week, is it ok?

Reply:
sure, just wanted to remind youu as this is from six months ago (and not something one would remember this long)